test/test_sparse.py

```diff
@@ -289,7 +288,7 @@ def test_tensor(x, res):
             def fn(x):
                 return x.to_dense()
             x.requires_grad_(True)
-            gradcheck(fn, (x,), check_sparse_nnz=True)
+            gradcheck(fn, (x,), check_sparse_nnz=True, check_batched_grad=False)
```

Comment:
Good idea! I don't think it would be too confusing, since the entire thing is designed to be used transparently. OTOH, seems like check_sparse_nnz should be True by default for the file as well but factoring that out into a `functools.partial` might be confusing.